### DISCUSSION TITLE
3.2.5に関する不具合や質問はこちら

### DISCUSSION THREAD

**Discussion by dongaba** [dongaba/TVerRec#129] [Issues]
### 3.2.5 Added

- Windows用Arm版ffmpegのダウンロード機能を追加

### 3.2.5 Changed

- 「放送分」だけでなく「配信分」からも日付を取得
- サイトマップとトップページにアクセスできない際のエラーを抑止
- TVerIDを登録している状態でもマイページ以外ではTVerIDの情報を使わないように変更
- 個別ダウンロード時に同時ダウンロード数の制御をするように変更 [Issue #124](https://github.com/dongaba/TVerRec/issues/124)

### 3.2.5 Fixed

- フリーワードサーチが失敗する問題を修正
- Windows GUI版で、個別ダウンロードがエラーとなる問題を修正 [Issue #125](https://github.com/dongaba/TVerRec/issues/125)

## What's Changed
* 3.2.5 Beta1 by @dongaba in https://github.com/dongaba/TVerRec/pull/123
* 3.2.5 Beta2 by @dongaba in https://github.com/dongaba/TVerRec/pull/126
* 3.2.5 Beta3 by @dongaba in https://github.com/dongaba/TVerRec/pull/127
* 3.2.5 by @dongaba in https://github.com/dongaba/TVerRec/pull/128

**Full Changelog**: https://github.com/dongaba/TVerRec/compare/v3.2.4...v3.2.5

<hr /><em>This discussion was created from the release <a href='https://github.com/dongaba/TVerRec/releases/tag/v3.2.5'>3.2.5</a>.</em>

**Comment by SilentBear555** [comment]
3.2.4から3.2.5にして、下記URLを試してみましたが、やはりエラーになります。
-----------------------------------------------------------------------------------------------
　バスVS鉄道22 勝てたルートを大公開！負けたチームのリーダーが悔やみまくり！
　https://tver.jp/episodes/eplz3bv0i9?p=1155
-----------------------------------------------------------------------------------------------
エラーメッセージは、以下の通りです。
　⚠️ エラーが発生しました。スキップして次のリンクを処理します。 
　- Response status code does not indicate success: 400 (Bad Request).
　⚠️ 番組情報を取得できませんでした。スキップします
-----------------------------------------------------------------------------------------------
なお、この動画hは、テレ東のサイトにも有り、そちらからから既にダウンロードできてますので
急いでバグ修正を行って頂く必要はないです。
ただ、以前から、ときどき、TVerRecでダウンロードできないURLが有るので、改善して頂けると有り難いです。

**Reply by dongaba** [reply]
`https://tver.jp/episodes/eplz3bv0i9?p=1155`から`?p=1155`の部分を除いたものが番組URLです。
`?p=1155`の部分はおそらく途中まで視聴した後の続きから再生のURLではないでしょうか?
`https://tver.jp/episodes/eplz3bv0i9`を指定すればダウンロードできると思います。

**Comment by SilentBear555** [comment]
ご指示の通りにURL指定して、無事ダウンロードできました。
おいそがしいところ、丁寧に対応して頂き、ありがとうございました。

**Comment by Harry-O** [comment]
3.2.5から初めて使わせていただいております。海外在住なので、たいへんありがたいツールです。作者様には感謝しかありません。

なお、一点問題が発生したので、回避策と合わせてご報告させていただきます。もし同じような問題に遭遇された方がいらした場合、ご参考になれば幸いです。

1.  3.2.5を新規DL後、リスト作成を実行
→ Cannot bind argument to parameter 'DifferenceObject' because it is null._
というメッセージが表示され、リストファイルに動画が追加されませんでした。数回試しましたが同じ結果です。

2. 3.2.4を別ディレクトリにDLし、リスト作成を実行
→ 問題なくリストに動画が追加されました。

3. 3.2.4で作成したリストを3.2.5のディレクトリにコピーし、3.2.5を起動してリスト作成を実行
→ 問題なくリストに動画が追加され、以降は3.2.5のみで問題なく使用できています。

以上です。

**Reply by dongaba** [reply]
ご報告ありがとうございます。
3.2.5でソースコードを修正した際に考慮が漏れていたようです。
次回のリリースで修正予定です。

**Reply by dongaba** [reply]
3.2.6で対応したつもりです
